sources/constraint-declaration-validation.asciidoc

```diff
@@ -876,7 +876,12 @@ public interface ParameterNameProvider {
 }
 ----
 
-[tck-testable]#A conforming Bean Validation implementation provides a default [classname]`ParameterNameProvider` implementation which returns parameter names in the form arg``PARAMETER_INDEX``, where `PARAMETER_INDEX` starts at 0 for the first parameter, e.g. [varname]`arg0`, [varname]`arg1` etc.#
+[tck-testable]#A conforming Bean Validation implementation provides a default [classname]`ParameterNameProvider` implementation which returns parameter names as stored in the class file containing the valided executable, if present. A conforming implementation must either use the Java reflection API or ensure behavioral compatability to using the reflection API in this way:
```

Comment:
Hmmm, I nearly fall into your trap! Looks like [tck-testable]# does not support being spanned on multiple lines like that. I would have fixed it on merging but I'm not sure of the best way to fix that as in this case, the items in the list are the important part.

Btw, s/compatability/compatibility/ (even if there is an entry in the wiktionary for compatability: https://en.wiktionary.org/wiki/compatability)